vars/sectionDeployToEnvironment.groovy

```diff
@@ -52,7 +52,10 @@ def call(params) {
             if (config.migrateDb) {
               stage("DB Migration - ${environment}") {
                 pcr.callAround("dbmigrate:${environment}") {
-                  builder.dbMigrate(tfOutput.vaultName.value, tfOutput.microserviceName.value)
+                  builder.dbMigrate(
+                    tfOutput.vaultName ? tfOutput.vaultName.value : config.dbMigrationVaultName,
+                    tfOutput.microserviceName.value
```

Comment:
hmm we need to do something about this as well so we can stop relying on the tfoutput,
shouldn't this just be the component parameter?